scripts/fast-reboot-dump.py

```diff
@@ -275,10 +281,17 @@ def send_garp_nd(neighbor_entries, map_mac_ip_per_vlan):
     # send arp/ndp packets
     for vlan_name, dst_mac, dst_ip in neighbor_entries:
         src_if = map_mac_ip_per_vlan[vlan_name][dst_mac]
+        if subtype and 'dualtor' in subtype.lower():
+            # for non dualtor devices, src mac will be router_mac
+            src_mac = src_mac_addrs[src_if]
+        else:
+            # for dualtor devices, the src mac will be vlan_mac
+            src_mac = get_iface_mac_addr(vlan_name)
```

Comment:
Isn't this the other way round?  This else case seems to be for non-dualtor.

Reply:
Absolutely, yes :) Fixed this in latest. Finding out a way to test this change.